### DISCUSSION TITLE
Can yabai function like how FancyZones does?

### DISCUSSION THREAD

**Discussion by DenizUgur** [asmvik/yabai#1294] [General]
I would love to define a custom grid layout and modify my windows exactly like FancyZones does. Does yabai offer this?

**Comment by asmvik** [comment]
I don't know much about FancyZones, but my initial take is no. You can run yabai in floating mode and set up keybinds to make windows take a specific position and size (using a grid to specify this information), but there is no snapping using the mouse in this mode.

**Reply by MuhammedZakir** [reply]
@_DenizUgur_: If you really want mouse/trackpad support, and don't mind using one more app, you can use
- (recommended) Hummingbird (mouse) or Penc (trackpad).
- (adventurous) Hammerspoon.
